diff --git a/Makefile b/Makefile
index 8ecdb72..07620ac 100644
--- a/Makefile
+++ b/Makefile
@@ -1,11 +1,12 @@
 EXTRA_CFLAGS += $(USER_EXTRA_CFLAGS)
 
 ##################### Optimization Flags ##########################
+EXTRA_CFLAGS += -Wno-error
 EXTRA_CFLAGS += -O2
 ##################### Compile-time Warnings #######################
-EXTRA_CFLAGS += -Wno-unused-variable
-EXTRA_CFLAGS += -Wno-unused-label
-EXTRA_CFLAGS += -Wno-unused-function
+# EXTRA_CFLAGS += -Wno-unused-variable
+# EXTRA_CFLAGS += -Wno-unused-label
+# EXTRA_CFLAGS += -Wno-unused-function
 
 GCC_VER_49 := $(shell echo `$(CC) -dumpversion | cut -f1-2 -d.` \>= 4.9 | bc )
 ifeq ($(GCC_VER_49),1)
